siliconcompiler/report/streamlit_report.py

```diff
@@ -22,6 +20,9 @@ class Dashboard():
     __port = 8501
 
     def __init__(self, chip, port=None):
+        if not _has_streamlit:
+            raise RuntimeError("Streamlit is not available")
+
```

Comment:
If we the user has the wrong version we could do something like this.
```suggestion
    if sys.version_info[0] < (3,7,0):
      raise RuntimeError("Dashboard is not available in 3.6")
```